include/fizzy/fizzy.h

```diff
@@ -32,10 +32,10 @@ typedef struct FizzyExecutionResult
 {
     /// Whether execution ended with a trap.
     bool trapped;
-    /// Whether function returned a value. Valid only if trapped equals false.
+    /// Whether function returned a value.
     bool has_value;
```

Comment:
So if we make this ambiguous here, why don't we state that if `trapped = false`, the other fields and undefined behaviour.

Reply:
I don't see this ambiguous. This is `false` if `trapped == true`. You can add this in the description.

Reply:
Added a note.